Lib/fontTools/merge.py

```diff
@@ -200,7 +222,7 @@ def merge(self, m, tables):
 	'macStyle': first,
 	'lowestRecPPEM': max,
 	'fontDirectionHint': lambda lst: 2,
-	'indexToLocFormat': recalculate,
+	'indexToLocFormat': first,
```

Comment:
I forgot the reason of this change, and what has to do with CFF, i thought head.indexToLocFormat is for loca, hence TT only

Reply:
https://github.com/fonttools/fonttools/pull/1289#discussion_r200960592

Reply:
ah ok thank you. I wonder if taking the first instead of recalculating could break things when attempting to merge TTFs that have different indexToLocFormat.

Reply:
Looks like `loca` table `compile()` always re-calculates it, so I don’t think it matters what value we set here.